docpipeline/generate.py

```diff
@@ -72,10 +74,24 @@ def clone_templates(dir):
     )
 
 
-def process_blob(blob, credentials, devsite_template):
-    log.info(f"Processing {blob.name}...")
+def format_docfx_json(metadata):
+    pkg = metadata.name
+    xrefs = ", ".join([f'"{xref}"' for xref in metadata.xrefs])
+    xref_services = ", ".join([f'"{xref}"' for xref in metadata.xref_services])
+
+    return DOCFX_JSON_TEMPLATE.format(
+        **{
```

Comment:
this seems a bit odd. Are we expanding this to keyword args? is there a reason to not just do this as such here then? I think this might be more of a copy paste though from before?

Reply:
I think it was an artifact of trying to figure out how to make it work. :sweat_smile: Switched to keyword args and it's much nicer. Thanks!